### ISSUE TITLE
Error on deployment of single function via `serverless deploy function -f functionName`

### ISSUE THREAD

**Issue by akleiber** [DataDog/serverless-plugin-datadog#85]
## Expected Behavior

Deploying a function via `node ./node_modules/.bin/serverless deploy -f functionName` should work.

## Actual Behavior

Deploying a function does not work and Serverless Framework will throw the following error after uploading the function zip.
See https://www.serverless.com/framework/docs/providers/aws/cli-reference/deploy-function/

```
Type Error ---------------------------------------------

  TypeError: Cannot read property 'Outputs' of undefined
      at Object.<anonymous> (/path/to/my/project/node_modules/serverless-plugin-datadog/src/output.ts:15:78)
      at Generator.next (<anonymous>)
      at fulfilled ((/path/to/my/project/node_modules/serverless-plugin-datadog/dist/src/output.js:5:58)
```

## Steps to Reproduce the Problem

  1. create serverless project
  1. add serverless-plugin-datadog plugin
  1. deploy stack
  1. change function and deploy single function

## Specifications

  - Serverless Framework version: 2.3.0
  - Datadog Serverless Plugin version: 2.6.2
  - Lambda function runtime (Python 3.7, Node 10, etc.): Node 12.x
  
## Stacktrace
  
```
Type Error ---------------------------------------------

  TypeError: Cannot read property 'Outputs' of undefined
      at Object.<anonymous> (/path/to/my/project/node_modules/serverless-plugin-datadog/src/output.ts:15:78)
      at Generator.next (<anonymous>)
      at fulfilled ((/path/to/my/project/node_modules/serverless-plugin-datadog/dist/src/output.js:5:58)
```

**Comment by tianchu** [comment]
@akleiber Thanks for reporting the issue! Do you mind trying again with the latest version of the Datadog serverless plugin? BTW, are you able to reproduce this issue on a hello-world application? if not, is there anything "unique" about your application that can help us reproduce the issue?

**Comment by jreyeshdez** [comment]
Hello,

I am also experiencing exact same issue with latest version of the DataDog serverless plugin.

## Specifications
* Serverless Framework version: 2.15.0
* Datadog Serverless Plugin version: 2.9.0
* Lambda function runtime (Python 3.7, Node 10, etc.): Go 1.15.5

**Comment by tianchu** [comment]
@jreyeshdez unfortunately I think the datadog serverless plugin only supports Python and Node.js at this moment.

**Comment by jreyeshdez** [comment]
@tianchu I am using the DataDog serverless plugin with Go no problem so far. Only issue I have seen currently is the one this ticket describes when trying to deploy single functions. 

There is no problem if deployed via `sls deploy` in Go as long as latest release from `datadog-lambda-go` is used since it support tracing: https://github.com/DataDog/datadog-lambda-go/releases/tag/v0.9.0

**Comment by akleiber** [comment]
Sorry I did not had the time to create a hello-world application.
We will move away from the plugin, because it is doing too much for us and we have another Lambda Layer that unfortunately does not run together with the datadog layer.

So you can either close this or keep open if others also experience this issue.
If we use the plugin again I will come back here and will provide more infos if it is not solved yet :-)

**Comment by markgibaud-vtail** [comment]
Hi @tianchu we're also getting this error, we're on:

Serverless Framework version: 2.11.1
Datadog Serverless Plugin version: 2.10.1
Lambda function runtime (Python 3.7, Node 10, etc.): Node 12.

Full deploy works but single function deploy results in:

```
Type Error ---------------------------------------------

  TypeError: Cannot read property 'Outputs' of undefined
      at Object.<anonymous> (/.../backend/node_modules/serverless-plugin-datadog/src/output.ts:15:78)
      at Generator.next (<anonymous>)
      at fulfilled (/.../backend/node_modules/serverless-plugin-datadog/dist/src/output.js:5:58)
```

**Comment by bryantbiggs** [comment]
we too are still seeing this error even on v2.12.0
<img width="953" alt="image" src="https://user-images.githubusercontent.com/10913471/104253307-26c5bd00-5442-11eb-94be-556c6e56399c.png">

**Comment by DarcyRaynerDD** [comment]
H all. 
We've included a PR from @jreyeshdez #96 in release v2.13.0 that addresses this issue. Let us know if it you run in to any problems.

**Comment by bryantbiggs** [comment]
confirmed as well - thank you all!
